src/sentry/relay/config/measurements.py

```diff
@@ -0,0 +1,47 @@
+""" Relay configuration related to transaction measurements. """
+
+
+from typing import Sequence, TypedDict
+
+#: The maximum number of custom measurements to be extracted from transactions.
+CUSTOM_MEASUREMENT_LIMIT = 5
+
+
+#: List used to distinguish between user-defined and well-known measurements.
+#: NOTE: This is redundant with `ALL_MEASUREMENT_METRICS`, which can be removed
+#: once all Relay instances understand the new format.
+BUILTIN_MEASUREMENTS = [
+    "app_start_cold",
+    "app_start_warm",
+    "cls",
+    "fcp",
+    "fid",
+    "fp",
+    "frames_frozen_rate",
+    "frames_frozen",
+    "frames_slow_rate",
+    "frames_slow",
+    "frames_total",
+    "inp",
```

Comment:
INP was recently added, see https://github.com/getsentry/sentry/pull/39010.

Reply:
#39010 is not merged yet, but I guess it's fine?

Reply:
Yep, should do no harm to already have it here.